packages/rollup-plugin-polyfills-loader/test/snapshots/flattened.html

```diff
@@ -34,9 +34,9 @@
   }
   function loadFiles() {
     [function () {
-      return loadScript('./entrypoint-a.js', 'module', []);
-    }, function () {
       return loadScript('./entrypoint-b.js', 'module', []);
+    }, function () {
+      return loadScript('./entrypoint-a.js', 'module', []);
```

Comment:
hmmm I'm just wondering is the order here maybe wrong?

as it's now `entrypoint-b` and then `entrypoint-a` 🤔